app/gui/view/graph-editor/src/builtin/visualization/native/raw_text.rs

```diff
@@ -147,6 +161,15 @@ impl RawTextModel {
     }
 }
 
+fn split_long_lines(data_str: &String, process_line: &mut impl FnMut(String) -> Result<(), DataError>) -> Result<(), DataError> {
+    let it = data_str.chars().chunks(1024);
```

Comment:
Actually, maybe the `1024` should be passed as a parameter to this method?

Reply:
Right. Done in [ef3a3a5](https://github.com/enso-org/enso/pull/3665/commits/ef3a3a520ebebcb110197a6a7e2c626849ce2672)